deps/streamsearch/sbmh.js

```diff
@@ -107,14 +117,11 @@ SBMH.prototype._sbmh_feed = function(data) {
       ) {
         this._lookbehind_size = 0;
         ++this.matches;
-        if (pos > 0)
```

Comment:
pos can never be bigger than 0 because we checked already in line 98, that pos is < 0